pypastry/commands/run.py

```diff
@@ -6,8 +6,8 @@
 
 def run():
     parser = argparse.ArgumentParser(prog='pastry run')
+    parser.add_argument('-m', '--message', required=True, type=str, help='Summary message about the experiment.')
```

Comment:
I think we probably don't need the message any more, as this was just to go in the git commit

Reply:
This message is stored in `Git summary` column (`data["git_summary"]` in the result JSON), so that user can still add some comments about his/her experiments (as git commit message is now gone)